x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_statistics.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiHealth } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useCnvmStatisticsApi } from '../../common/api/use_vulnerabilities_stats_api';
+import { useNavigateVulnerabilities } from '../../common/hooks/use_navigate_findings';
+import { CompactFormattedNumber } from '../../components/compact_formatted_number';
+import { getSeverityStatusColor } from '../../common/utils/get_vulnerability_colors';
+import { CspCounterCard } from '../../components/csp_counter_card';
+
+export const VulnerabilityStatistics = () => {
+  const navToVulnerabilities = useNavigateVulnerabilities();
+  const getCnvmStats = useCnvmStatisticsApi();
+
+  const stats = useMemo(
+    () => [
+      {
+        id: 'cloud-regions-stat',
+        title: <CompactFormattedNumber number={getCnvmStats.data?.cnvmStatistics.cloudRegions} />,
+        description: i18n.translate('xpack.csp.cnvmDashboard.statistics.cloudRegionTitle', {
+          defaultMessage: 'Cloud Regions',
+        }),
+      },
+      {
+        id: 'assets-scanned-stat',
+        title: (
+          <CompactFormattedNumber number={getCnvmStats.data?.cnvmStatistics.resourcesScanned} />
+        ),
+        description: i18n.translate('xpack.csp.cnvmDashboard.statistics.assetsScannedTitle', {
+          defaultMessage: 'Assets Scanned',
```

Comment:
```suggestion
          defaultMessage: 'Resources Scanned',
```

It was discussed in the figma comments but wasn't updated in the figma itself